x-pack/heartbeat/monitors/browser/synthexec/execmultiplexer_test.go

```diff
@@ -33,8 +33,9 @@ func TestExecMultiplexer(t *testing.T) {
 
 		for sIdx := 0; sIdx < 3; sIdx++ {
 			step := &Step{
-				Name:  fmt.Sprintf("S%d", sIdx),
-				Index: sIdx,
+				Name:   fmt.Sprintf("S%d", sIdx),
+				Index:  sIdx,
+				Status: fmt.Sprintf("failed%d", sIdx),
```

Comment:
Why do we need to interpolate the step number into the status? The status field is an enum. The reason we interpolate the index into the step name is to give unique step names, but that doesn't apply here.

Reply:
fixed !!